content/aips/ACI_SERVICE_PROVIDER_PROPOSAL.md

```diff
@@ -0,0 +1,63 @@
+---
+title: ACI Service Provider Proposal
+author: Marc Zeller (@marczeller), Aave-Chan initiative
+shortDescription: ACI Service Provider Proposal
+discussions: https://governance.aave.com/t/arfc-aci-service-provider-6-month-proposal/12513
+created: 2023-04-12
+---
+
+## Simple Summary
+
+This AIP proposal presents Aave with the opportunity to onboard Aave-Chan Initiative (ACI) as a service provider for the Aave protocol.
+
+## Motivation
+The Aave Chan Initiative has been a dedicated contributor to the Aave ecosystem. The 6-Month Budget Proposal aims to secure the necessary resources to continue the ACI's work, ensuring that we can maintain our level of commitment and contribution to the Aave ecosystem. For more details about the ACI proposal, please refers to the [TEMP CHECK](https://governance.aave.com/t/temp-check-aave-chan-initiative-6-month-budget-proposal/12344) version of the current proposal
+
+## Specification
+If this proposal is approved, a stream of the allocated budget will be activated, with [treasury.aavechan.eth](https://etherscan.io/address/0x57ab7ee15cE5ECacB1aB84EE42D5A9d0d8112922) as the recipient.
+
+This will allow the ACI to periodically claim a fraction of the budget for the duration of the stream.
+
+The proposed budget is 250k aUSDT for a period of 6 months.
+
+The proposal payload uses the `IProposalGenericExecutor` to create a 180 days stream with the following parameters:
```

Comment:
```suggestion
The proposal payload uses the delegatecall pattern with a pre-deployed payload. The payload follows the `IProposalGenericExecutor` interface to create a 180 days stream with the following parameters:
```

`IProposalGenericExecutor` is just an interface forcing you to have an `execute()` function